### PR TITLE
Application Status History

### PR DESCRIPTION
**Pull request by srleach**
## Features

- Add view to basket for Application Status History

Delivers [#126577047](https://www.pivotaltracker.com/story/show/126577047)

### CODE REVIEW

**Comment by srleach** [comment]
@EvanBarbour3 Pre review please

File: resources/views/applications/show.blade.php [hunk 1]
```diff
@@ -6,7 +6,7 @@
         <div class="btn-group pull-right">
             <a href="{{Request::url()}}/fulfil" class="btn btn-info{{ $fulfilmentAvailable == true ? ' ' : ' disabled' }}"><span class="glyphicon glyphicon-gift"></span> Fulfil</a>
             <a href="{{Request::url()}}/request-cancellation" class="btn btn-danger{{ $cancellationAvailable == true ? ' ' : ' disabled' }}"><span class="glyphicon glyphicon-remove-circle"></span> Request Cancellation</a>
-            @if(Auth::user()->can('applications-merchant-payments'))<a href="{{Request::url()}}/add-merchant-payment" class="btn btn-success"><span class="glyphicon glyphicon-plus-sign"></span> Add Merchant Payment</a>@endif
+            @if(Auth::user()->can('applications-merchant-payments'))<a href="{{Request::url()}}/add-merchant-payment" class="btn btn-success{{ $merchantPaymentsAvailable == true ? ' ' : ' disabled' }}"><span class="glyphicon glyphicon-plus-sign"></span> Add Merchant Payment</a>@endif
```
Comment (EvanBarbour3) [inline resources/views/applications/show.blade.php]: 👍

**Comment by EvanBarbour3** [comment]
@wnowicki Happy, pending scrutinizer

**Comment by wnowicki** [comment]
👍 waiting for CI

**Comment by wnowicki** [comment]
- one major issue introduced

**Comment by srleach** [comment]
@wnowicki 

```scrutinizer
The method getApplicationHistory() does not exist on PayBreak\Sdk\Gateways\ApplicationGateway. Did you maybe mean getApplication()?
```

SDK Isn't released yet, that's why ;)

**Pull request merged**